### ISSUE TITLE
Tutorial 2 Import Error

### ISSUE THREAD

**Issue by stoneandstone** [flow-project/flow#756]
## Bug Description

In tutorial 2, when i run python scrip ' from flow.networks.ring import RingNetwork', system return a importerror.  At beginning i thought conda environment is not suitable. I checked the conda environment packages and file 'environment.yml' , i think it is ok, every package is correctly installed. lxml version is 4.2.4 , same as environment.yml . 

## Bug Reproduce

File in which the bug occurred: `file.py`

The exact command that you did run that caused the bug:
`command`

## Output Log

Full error output log:

```
ImportErrorTraceback(most recent call last)
<ipython-input-2-9914ac7a3512>in <module>
---->1 from flow. networks. ring import RingNetwork
~/flow/flow/networks/_init. py in <module>
2
3 # base network class
---->4 from flow. networks. base import Network
5
6 # custom networks
~/flow/flow/networks/base. py in <module>
7 import time
8 import xml. etree. ElementTree as ElementTree
---->9from lxml import etree
10 from collections import defaultdict
11
```

```
ImportError: dlopen(//anaconda3/envs/flow/lib/python3.6/site-packages/xml/etree. cpython-36m-darwin. so,2): no suitable image found. Did find:
//∥anaconda3/envs/flow/lib/python3.6/site-packages/lxml/etree. cpython-36m-darwin. so: unknown file type, first eight bytes:0x00 0x00 0x00 0x00 0×00 0×00 0×00 0×00
/anaconda3/envs/flow/lib/python3.6/site-packages/lxml/etree. cpython-36m-darwin. so: unknown file type, first eight bytes:0x00 0x00 0x00 0×00 0x00 0x00 0x00 0×00
```

## Screenshots

Screenshots:

![picture name](https://link/to/screenshot)

## Versions

- OS: macOS
- Flow version: latest version
- Flow commit number: ? (run `git log` and copy the commit number)

**Comment by eugenevinitsky** [comment]
@kevin-thankyou-lin is there a chance you could take a look at this and see if you can reproduce it?

**Comment by kevin-thankyou-lin** [comment]
Yeah, for sure. I don't have the license for Aimsun, so I don't think I can run everything in tutorial 2. That said, the line 

"from flow. networks. ring import RingNetwork"

works fine in my "flow" conda environment. 

@stoneandstone If you haven't already, I'd check out this page: https://github.com/flow-project/flow/blob/master/tutorials/README.md

Specifically, I'd check if any of this fixes the import error:

"
Common error: if, when running a notebook, you run into an error of the form ImportError: No module named flow.something, this probably means that the flow Conda environment is not active in your notebook. Go into the Conda tab and make sure that flow is selected. In case you don't have this Conda tab, try running conda install nb_conda just after source activate flow, then open the notebook again. If this doesn't work either, you can try other solutions here , or you can launch a Jupyter notebook using the flow environment directly from the Anaconda Navigator. If you have the conda tab but you still get the error, open a new terminal and execute the following commands:

    cd /
    source activate flow
    python

    (cd / is to make sure that flow is not in the folder you run python from). 

Then, in the Python interface that opens, run import flow. If you get an ImportError, this means you haven't installed Flow in your environment. Go back to the installation instructions, especially the part where you do pip install -e . after having done source activate flow.
"

**Comment by wwxb2012** [comment]
You need 'conda update --all'
